README.md

```diff
@@ -0,0 +1,132 @@
+# TABObserverSet
+
+`TABObserverSet`, originally conceived by [Mike Ash](https://github.com/mikeash/SwiftObserverSet), provides a Swift-y alternative to the traditional `NotificationCenter` style of reactive programming.
+
+With a simple syntax, `TABObserverSet` is easy to use and read in your code.
+
+## Usage
+
+Usage is very simple. Similar to `NotificationCenter`, you have a _broadcaster_ and _observers_. While `NotificationCenter`-style broadcasting can potentially result in multiple _broadcasters_ and multiple _observers_, `TABObserverSet` results in a single _broadcaster_  and multiple _observers_.
```

Comment:
Normally the language around this is defining the relationship as one to many or many to one or many to many. It might be worth adopting this style